securedrop_client/gui/widgets.py

```diff
@@ -382,6 +382,9 @@ def update_message(self, message: str, duration: int) -> None:
         continuously show message.
         """
         self.status_bar.showMessage(message, duration)
+        new_width = self.fontMetrics().horizontalAdvance(message)
+        self.status_bar.setMinimumWidth(new_width)
+        self.status_bar.reformat()
```

Comment:
This does fix the issue where we hide part of the "reconnect..." word when the client is minimized to its smallest window size, but after reading this code I was expecting this change to fit the white error status bar to the message text (maybe it does on your system?) For example you can see what I'm seeing with "Failed to update star.": 
![Screenshot from 2020-03-23 11-32-22](https://user-images.githubusercontent.com/4522213/77351593-37624680-6cfb-11ea-944c-b830db104024.png)

Reply:
I've looked into this and, while not 100% certain, I believe the following is going on:

When the `status_bar` is added to the parent, it (and the icon and red bar to its left) are also surrounded by padding to presumably ensure it remains properly centred.

I can only assume that the behaviour you see is simple the outcome of such an interaction between the width defined by the padding I just mentioned and the width as specified by my code. The outcome is, I guess, that there's a minimal width.

Here's where we get pragmatic. I could spend much more time on this, but would it be worth it? The text is guaranteed to be displayed by my change, and shorter text may have a larger width (because of the padding behaviour) but that will be of a uniform width, as far as I can tell.

Happy to change things around, but I don't want to dive into a rabbit hole if it's not necessary. :+1: